crates/ruma-common/src/events/secret_storage/some_secret.rs

```diff
@@ -0,0 +1,34 @@
+//! Types for the *m.secret_storage.some_secret* event.
```

Comment:
I don't see this in the spec and it looks very much like a placeholder. Are you sure this module should exist at all?

Reply:
You're right that event was definitely a placeholder. I've removed the module, but not sure where to put `AesHmacSha2EncryptedData` which I think was the only useful part of this file. Currently I left it in `events/secret_storage.rs`.

Reply:
Actually, I think I should make this an event `#[ruma_event(type = "m.secret_storage.key.*", kind = GlobalAccountData)]`.
It also has varying properties depending on the algorithm used, so I'll give it a more generic name (maybe `EncryptedSecretData`?) and make it work like `KeyDescription`.